google-cloud-bigtable/lib/google/cloud/bigtable/table.rb

```diff
@@ -168,13 +186,40 @@ def cluster_states
         #   # Get a column family by name
         #   cf1 = table.column_families["cf1"]
         #
+        # @example Modify the table's column families
+        #   require "google/cloud/bigtable"
+        #
+        #   bigtable = Google::Cloud::Bigtable.new
+        #
+        #   table = bigtable.table("my-instance", "my-table", perform_lookup: true)
+        #
+        #   table.column_families do |cfs|
```

Comment:
Maybe call the block param something like "cfupdater", to emphasize the type.

Reply:
I can do that. `cfs` was already used in the `create_table` block examples, so I just stayed with it.

Reply:
Also, the lines in this example are long, the variable name needs to be something short.

Reply:
True, but previously the object was a collection of ColumnFamilies, so the name made sense. Now we've changed the type to be an updater object (not a collection).

Reply:
Replaced `cfs` and `column_families` with `cf_creator` and `cf_updater`.